UNRELEASED.md

```diff
@@ -23,6 +23,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Documentation
 
 - Added `Stack.Item` properties and description to [style guide](https://polaris.shopify.com)’s ([#772](https://github.com/Shopify/polaris-react/pull/772))
+- Added accessibility documentation to the `Resource list` and `Data table` components ([#927](https://github.com/Shopify/polaris-react/pull/927))
```

Comment:
Referring to the React components?
```
- Added accessibility documentation to the `ResourceList` and `DataTable` components ([#927](https://github.com/Shopify/polaris-react/pull/927))
```

Referring to the components as a general concept?

```
- Added accessibility documentation to the resource list and data table components ([#927](https://github.com/Shopify/polaris-react/pull/927))
```

Reply:
@kaelig I've seen both formats, as well as what I did. 😆Which is preferred?

Reply:
In that case, I think this would fit (Referring to the components as a general concept):

```suggestion
- Added accessibility documentation to the resource list and data table components ([#927](https://github.com/Shopify/polaris-react/pull/927))
```